joomla/joomla-cms
# [Install Languages] Add logo layer on install languages packages

Pull Request for Improvement.
#### Summary of Changes

Installations on backend use a layer with an animated Joomla! logo when installing something.
All except one. The install languages.

This PR adds the logo layer when installing languages too.

![image](https://cloud.githubusercontent.com/assets/9630530/15179527/48d007c6-1774-11e6-9527-4e855d5576b3.png)
#### Testing Instructions
1. Use latest staging, apply patch and clean browser cache
2. Go to Extensions -> Manage -> Install Languages
3. Install one or several languages.<br/>You will now see joomla logo when installing the languages packages.
